pkg/webui/components/map/widget/index.js

```diff
@@ -73,13 +73,14 @@ export default class MapWidget extends React.Component {
   }
 
   render() {
-    const { path } = this.props
+    const { path, markers } = this.props
 
     return (
       <WidgetContainer
         title={sharedMessages.location}
         toAllUrl={path}
         linkMessage={sharedMessages.changeLocation}
+        disableLink={markers && markers.length > 0}
```

Comment:
This solution is a bit ambiguous. `disableLink` suggests that it disables all links (also the one in the top right hand corner). Actually, I think it's better to move the wrapping link out of the widget container to the passed child prop directly. That gives back full control over the behavior of the wrapped content.